Automattic/newspack-plugin
# feat: update wizard footer logo with WordPress.com and Google

### All Submissions:






### Changes proposed in this Pull Request:

Replaces the Newspack icon located in the footer by the WordPress.com logo and the Google logo.

![Screen Shot 2020-02-25 at 16 56 35](https://user-images.githubusercontent.com/177929/75269527-f573dc80-57f0-11ea-94a9-8bb3b814626f.png)
![Screen Shot 2020-02-25 at 16 59 26](https://user-images.githubusercontent.com/177929/75269530-f6a50980-57f0-11ea-8f72-1b96aa80a8fe.png)


### How to test the changes in this Pull Request:

1. Switch to this branch.
2. Check any Wizards (inc. Setup - Welcome), you should see the new logos at the bottom

### Other information:
